tests/DataFactories/Settings.php

```diff
@@ -56,4 +57,16 @@ function withSkippedTutorials() {
     $this->settings->set('show_congratulate_after_first_newsletter', 0);
     return $this;
   }
+
+  function withSendingMethod($sending_method) {
+    $this->settings->set('mta.method', $sending_method);
+    $this->settings->set('mta_group', $sending_method === Mailer::METHOD_SMTP ? 'smtp' : 'website');
+  }
```

Comment:
Should this method be chainable like others?

Reply:
Definitely, will fix.